File: plugins/capawesome-capacitor-wifi/android/src/main/java/io/capawesome/capacitorjs/plugins/wifi/classes/results/IsEnabledResult.java
package io.capawesome.capacitorjs.plugins.wifi.classes.results;

import com.getcapacitor.JSObject;
import io.capawesome.capacitorjs.plugins.wifi.interfaces.Result;

public class IsEnabledResult implements Result {

    private boolean isEnabled;

    public IsEnabledResult(boolean isEnabled) {
        this.isEnabled = isEnabled;
    }

    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("enabled", isEnabled);
        return result;
    }
}

File: plugins/capawesome-capacitor-wifi/ios/Plugin/Classes/Results/IsEnabledResult.swift
import Foundation
import CoreBluetooth
import Capacitor

@objc public class IsEnabledResult: NSObject, Result {
    private let enabled: Bool

    init(enabled: Bool) {
        self.enabled = enabled
    }

    public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["enabled"] = enabled
        return result as AnyObject
    }
}
